manager2/src/app/admin/projects/projects.component.ts

```diff
@@ -147,10 +147,6 @@ export class ProjectsComponent implements OnInit {
         // about order, see: https://medium.com/@lukaonik/how-to-fix-the-previous-ngmodelchange-previous-value-in-angular-6c2838c3407d
         this.new_project.id = tmpprojectid; // todo: maybe add an option to enable or disable this one
 
-        if (!this.new_project.expire) {
-            this.new_project.expire = this.date_convert(new Date().getTime() + this.config.project.default_expire * this.day_time)
-        }
-
```

Comment:
Why not keep this as a fallback if expire is not set (old pending projects), instead of being an empty string?

Reply:
You're right, I brought it back.
While I'm at it, may I ask if [this](https://github.com/rsiminel/genouestaccountmanager/blob/56aca6f198bf2c5bce5a26d2099373424f103ff5/manager2/src/app/admin/project/project.component.ts#L92) is a sanctioned use of date_convert ?

Reply:
Not sure what you mean by 'sanctioned use' ?

Reply:
Sorry, I was unclear. I meant to ask why the function is being called there and whether it might cause the date to be stored as a string in the database after that function call.

Reply:
I think 'show_project_users' is in charge of actually getting the project. Since we show the expiration date in a editable field, I believe it need to be converted from DB format in that step.

We convert it back before updating [here](https://github.com/genouest/genouestaccountmanager/blob/364e137b3138e132539f3965618286ed0d179332/manager2/src/app/admin/project/project.component.ts#L164)

Reply:
Oh ok, thanks for explanation. I guess we can merge now?